__tests__/app.test.js

```diff
@@ -52,3 +53,51 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("returns 200 status code and the specified article by ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toHaveProperty("article");
+        const article = response.body.article;
+
+        expect(article.title).toBe("Living in the shadow of a great man");
```

Comment:
I think it would be good to add an expect statement here to ensure the article has the correct article id